pytorch/serve
# Remove model registered through URL from model store on unregister

## Description

Remove model registered through URL from model store on unregister.

Fixes #160 

## Type of change

Please delete options that are not relevant.


## Feature/Issue validation/testing
1) 
 - check the squeezenet1_1.mar file is not already available in model store

 - Register squeezenet model from S3 URL :
https://torchserve.s3.amazonaws.com/mar_files/squeezenet1_1.mar

 - Verify the model archive is downloaded in model_store directory

 - Unregister the squeezenet model
   
 - Verify the model archive is removed from model_store directory
 
2) Register and unregister a model from a mar file already present in model store. Validate that the mar file is not removed from the model store on unregistering the model.

- UT/IT execution results
[frontend_build_log_issue_160.txt](https://github.com/pytorch/serve/files/4444475/frontend_build_log_issue_160.txt)

## Checklist:
